src/cns/CnsProxyReader.ts

```diff
@@ -17,11 +17,20 @@ export default class CnsProxyReader implements ICnsReader {
     return this.get(tokenId);
```

Comment:
I don't think we need this class entirely now: it can just be a part of `Cns`.

Reply:
Agree, I removed the class and moved the logic into CNS. Had to remove the logic from EtheriumNamingService since the resolver function is now working slightly differently.